tests/e2e/basic/client_connectivity_test/test_vlan_mode.py

```diff
@@ -367,9 +373,10 @@ def test_modify_ssid(self, request, update_ssid, get_lanforge_data, setup_profil
             instantiate_testrail.update_testrail(case_id=test_cases["vlan_ssid_update"], run_id=instantiate_project,
                                                  status_id=1,
                                                  msg='5G WPA Client Connectivity Passed successfully - vlan mode '
-                                                     'updated ssid')
+                                                     'updated ssid' + str(run_results))
         else:
             instantiate_testrail.update_testrail(case_id=test_cases["vlan_ssid_update"], run_id=instantiate_project,
```

Comment:
I thought we were removing reference to testrail?

Reply:
yes i added this for debugging purpose, so in this way i could know what are the results, like detailed results, that we can see about these cases in testrails

Reply:
Hopefully we can use allure for that!